test.sh

```diff
@@ -76,6 +77,25 @@ if [[ -z $KAGGLE_KEY && ! ( -r "$HOME/.kaggle/kaggle.json" ) ]]; then
 fi
 
 set -x
+
+if [[ $NOTEBOOK == "all" ]]; then
+    docker run --rm -t \
+    -e KAGGLE_USERNAME -e KAGGLE_KEY \
+    -v ~/.kaggle:/root/.kaggle:ro \
+    -v $PWD:/input:ro \
+    $PINNED_IMAGE \
+    /bin/bash -c "/input/notebooks/test.sh kerasExp"
+fi
+
+if [[ $NOTEBOOK  == "computer_vision" || $NOTEBOOK == "deep_learning_intro" ]]; then
```

Comment:
I think this is not quite right, because after this block executes the test under the pinned image, the next block will execute it again under the staging image?  I think you need to `exit` at the end of this block.

Reply:
sorry about that, tbh i never thought i would ever need to write a bash script after college 🤣 .